app/ai/voice/agents/breeze_buddy/types/models.py

```
@@ -1,21 +1,40 @@
-    template: str
+    template: Optional[str] = None
+    template_id: Optional[str] = None
+
+    @field_validator("template_id")
+    @classmethod
+    def validate_template_id_format(cls, v: Optional[str]) -> Optional[str]:
+        if v is not None:
+            try:
+                UUID(v)
+            except ValueError:
+                raise ValueError("template_id must be a valid UUID")
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)